Spec/Tests/RestClientChannelTests.swift

```diff
@@ -300,7 +300,7 @@ class RestClientChannelTests: XCTestCase {
         let client = AblyTests.newRealtime(options)
         defer { client.dispose(); client.close() }
         waitUntil(timeout: testTimeout) { done in
-            client.channels.get("RSA7e1")
+            client.channels.get(uniqueChannelName(prefix: "RSA7e1"))
```

Comment:
I just decided to leave it as "tag", since the original test didn't use standard name "test" probably for a reason. So I've decided to add an additional recognition to where author of the original test did.